ecpy_hqc_legacy/tasks/tasks/instr/spdev_tasks.py

```diff
@@ -14,7 +14,7 @@
 
 import numbers
 import numpy as np
-from atom.api import (Bool, Unicode, Enum, set_default)
+from atom.api import (Bool, Unicode, Enum, set_default, Int)
```

Comment:
Int appears to be unused, please remove it.

Reply:
ok